webuni/commonmark-bundle/0.2/etc/packages/webuni_commonmark.yaml

```diff
@@ -0,0 +1,6 @@
+webuni_commonmark:
+    extensions:
+        # adds support for defining attributes on HTML elements (e.g. 'id', 'class', 'style')
+        attributes: true
+        # adds support for parsing Markdown tables
+        table: true
```

Comment:
I think the change should be done in the bundle itself (and doc should be contributed there too btw)